Gemfile

```diff
@@ -72,3 +72,5 @@ gem 'lefthook', '~> 1.6', '>= 1.6.10'
 gem 'devise', '~> 4.9', '>= 4.9.4'
 
 gem 'image_processing', '>= 1.2'
+
+gem 'kaminari'
```

Comment:
You're missing the version. Remember that the RubyGems website is a good place to find the versions.